src/index.js

```diff
@@ -1,11 +1,12 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
-import App from "./components/App/App";
+//import App from "./components/App/App";
```

Comment:
:eyes:

Reply:
Why did you comment the app component? @alejandro-castellon-jalafund